packages/cli-tool/src/template/fetchingStrategy/copyStrategy.ts

```diff
@@ -0,0 +1,36 @@
+import { CliUx } from '@oclif/core';
+
+import { FetchStrategy } from '.';
+import { InitTemplateOptions } from '../.';
+import runCommand from '../../helpers/child-process';
+
+class CopyStrategy implements FetchStrategy {
+  constructor(public selectedTemplate: string) {
+    this.selectedTemplate = selectedTemplate;
+  }
+
+  async fetchTemplateFiles(options: InitTemplateOptions): Promise<void> {
+    return this.copyTemplateFiles(options)
+      .then(() => this.renameFolder(options));
+  }
+
+  private async copyTemplateFiles(options: InitTemplateOptions): Promise<void> {
+    CliUx.ux.info('Copying template source files...');
```

Comment:
```suggestion
    CliUx.ux.info('Copying ${selectedTemplate} template source files...');
```

Should/Can we add the template name in the log? 👀